lib/src/fields/form_builder_signature_pad.dart

```diff
@@ -135,35 +140,50 @@ class _FormBuilderSignaturePadState extends State<FormBuilderSignaturePad> {
                 decoration: BoxDecoration(
                   border: Border.all(color: Colors.grey),
                 ),
-                child: GestureDetector(
-                  onVerticalDragUpdate: (_) {},
-                  child: Signature(
-                    width: widget.width,
-                    height: widget.height,
-                    backgroundColor: widget.backgroundColor,
-                    controller: _effectiveController,
-                  ),
-                ),
+                child: _savedValue != null || _readOnly
+                    ? Card(
```

Comment:
Cards have a 3-D effect -- are you sure you want that and not a plain Container?

Reply:
The idea is to distinguish between an editable and a non-editable signature pad. The editable pad is a text box-like container, whereas Card looks non-editable by default.